app/views/foreman_salt/job_templates/salt_run_command_-_salt_default.erb

```diff
@@ -0,0 +1,19 @@
+<%#
+name: Salt Script Command - Salt default
+job_category: Salt state
```

Comment:
Can you please change to job category "Salt" because e.g. https://github.com/theforeman/foreman_salt/blob/master/app/views/foreman_salt/job_templates/salt_run_state_highstate_-_ssh_default.erb is already using "Salt"